src/utils/helpers/propCustomValidation.js

```diff
@@ -0,0 +1,8 @@
+export const checkDateString = (props, createdTime) => {
+  const date = Date.parse(props[createdTime]);
+  if (isNaN(date)) {
```

Comment:
this is not a correct way of checking a date. prefer using a library function to validate a date

// see https://date-fns.org/v2.29.3/docs/isValid

```
new Date(Date.parse('0'))
-> Sat Jan 01 2000 00:00:00 GMT+0600 (Bangladesh Standard Time)
```